go.mod

```diff
@@ -189,6 +189,7 @@ require (
 )
 
 replace (
+	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```

Comment:
thanks, maybe better to change this in separate PR.

Reply:
yess